sdk/firstload/src/index.tsx

```diff
@@ -14,21 +14,24 @@ import {
 	Metric,
 	OnHighlightReadyOptions,
 	SessionDetails,
-} from '@highlight-run/client/src/types/types'
-import { MixpanelAPI, setupMixpanelIntegration } from './integrations/mixpanel'
+} from '@highlight-run/client/src/types/types.js'
+import {
+	MixpanelAPI,
+	setupMixpanelIntegration,
+} from './integrations/mixpanel.js'
 
-import { FirstLoadListeners } from '@highlight-run/client/src/listeners/first-load-listeners'
-import { GenerateSecureID } from '@highlight-run/client/src/utils/secure-id'
-import { HighlightSegmentMiddleware } from './integrations/segment'
-import configureElectronHighlight from './environments/electron'
+import { FirstLoadListeners } from '@highlight-run/client/src/listeners/first-load-listeners.js'
+import { GenerateSecureID } from '@highlight-run/client/src/utils/secure-id.js'
+import { HighlightSegmentMiddleware } from './integrations/segment.js'
+import configureElectronHighlight from './environments/electron.js'
 import firstloadVersion from './__generated/version'
 import {
 	SessionData,
 	getPreviousSessionData,
-} from '@highlight-run/client/src/utils/sessionStorage/highlightSession'
+} from '@highlight-run/client/src/utils/sessionStorage/highlightSession.js'
 import { initializeFetchListener } from './listeners/fetch'
 import { initializeWebSocketListener } from './listeners/web-socket'
-import { listenToChromeExtensionMessage } from './browserExtension/extensionListener'
+import { listenToChromeExtensionMessage } from './browserExtension/extensionListener.js'
```

Comment:
how does this work for the files that are imported? ie. `./listeners/fetch/index.ts` imports more files without a `.js` extension

Reply:
I just added `*/index.js` to these imports and it works great in both `node` and `nodenext`.